metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -320,3 +343,16 @@ def delete_one_urn_cmd(
         cached_session_host,
         cached_emitter,
     )
+
+
+def _delete_references(
+    urn: str,
+    dry_run: bool = False,
+    cached_session_host: Optional[Tuple[sessions.Session, str]] = None,
+) -> Tuple[int, List[Dict]]:
+    payload_obj = {"urn": urn, "dryRun": dry_run}
+    return cli_utils.post_delete_references_endpoint(
+        payload_obj,
+        "/entities?action=deleteReferences",
```

Comment:
Why exactly does the caller have to provide the "path"? Doesn't the method name imply that it should know which path to use?

Reply:
I'm not sure this comment applies the python logic. Are you talking about the code in java land that modifies the aspect?

Reply:
no- why do we need to provide this argument: `"/entities?action=deleteReferences",`. Shouldn't the method inside know where to request?

Reply:
I see what you mean. Yes I think it should, I kept it like this for consistency with other methods, i.e: `_delete_one_urn` also in this delete_cli.py file.

Do you think it makes sense to change this in a consistent way in this PR?